src/shared/components/catalog/utils/types.ts

```diff
@@ -65,6 +65,13 @@ export type CatalogItem<T extends any = any> = {
   // This description will also be shown in the side panel if there are no `details.descriptions`.
   description?: string | React.ReactNode;
   tags?: string[];
+  projectType?: string;
+  language?: string;
+  git?: {
+    remotes: {
+      origin: string;
+    };
+  };
```

Comment:
Do we need all this for every `CatalogItem`? This could be used for future catalogs in HAC as well. i think attributes could be used to store custom data in catalog items.